src/dmd/link.d

```diff
@@ -34,6 +35,23 @@ version (Posix) extern (C) int pipe(int*);
 version (Windows) extern (C) int spawnlp(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnl(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnv(int, const char*, const char**);
+version (Posix) extern (C) extern __gshared char** environ;
+version (CRuntime_Glibc)
+{
+    // borrowed from https://github.com/dlang/druntime/commit/51a9962482a668fab00cc866045d76a9c4a39b63
+    // remove and replace with core.sys.posix.spawn once 2.083 is bootstrap
+    extern(C):
+    struct posix_spawn_file_actions_t { void[80] pad; } // idk what is even in here but the sizeof matches
```

Comment:
unrelated to this PR but I don't know why this was added. extern C struct are usually body less as you work on them as pointer, so the size doesn't matter.

Reply:
You need to allocate memory for the struct on the stack. There's no other way to get a value of this type.